presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSplitSource.java

```diff
@@ -91,6 +91,8 @@ public void close()
     {
         try {
             closer.close();
+            this.fileScanTaskIterable = CloseableIterable.empty();
```

Comment:
The fileScanTaskIterable was actually not used. @wypb Do you know why it was added at the first place?

Reply:
Yes, this object is indeed not used and can be deleted. But I'm not sure why the author added `fileScanTaskIterable` objects, maybe it was accidentally added.